.circleci/integration-tests/master_dag.py

```diff
@@ -29,9 +30,12 @@ def get_report(dag_run_ids: List[str], **context: Any) -> None:
 
         airflow_version = context["ti"].xcom_pull(task_ids="get_airflow_version")
         airflow_executor = context["ti"].xcom_pull(task_ids="get_airflow_executor")
-        airflow_version_message = (
-            f"The below run is on Airflow version `{airflow_version} with {airflow_executor} executor`\n\n"
-        )
+
+        if IS_RUNTIME_RELEASE:
+            airflow_version_message = f"Results generated for latest Runtime version {os.environ['ASTRONOMER_RUNTIME_VERSION']} with {airflow_executor}  \n\n"
```

Comment:
```suggestion
            airflow_version_message = f"Results generated for Runtime version {os.environ['ASTRONOMER_RUNTIME_VERSION']} with {airflow_executor}  \n\n"
```
perhaps this may not be the latest runtime version always, no? Anyway good to skip the work latest since we're reflecting the version anyhow.

Reply:
correct @pankajkoti